chromium_src/chrome/browser/ui/views/overlay/video_overlay_window_views.cc

```diff
@@ -6,15 +6,18 @@
 #include "brave/browser/ui/views/overlay/brave_back_to_tab_label_button.h"
 #include "brave/browser/ui/views/overlay/brave_video_overlay_window_views.h"
 
+// Also has WrapUnique.
+#include "cc/trees/task_runner_provider.h"
+#include "content/public/browser/web_contents_user_data.h"
+
 #define BRAVE_UPDATE_MAX_SIZE max_size_ = work_area.size();
-#define BRAVE_CREATE        \
-  overlay_window->Close();  \
-  overlay_window.release(); \
-  overlay_window = std::make_unique<BraveVideoOverlayWindowViews>(controller);
+#define WrapUnique(VideoOverlayWindowViews) \
```

Comment:
I'm worring that this define seems fragile in the future as it's commly used method.
Then, it could make chromium bump more complex.
IMO, patching is more better in this case. WDYT?

Reply:
Yup good call